src/main/java/io/clue2solve/aws/bedrock/springboot/starter/config/ClaudeProperties.java

```diff
@@ -9,6 +9,15 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Sample properties aws.bedrock.model.claude.id=anthropic.claude-v2
+ * aws.bedrock.model.claude.prePrompt=Human:
+ * aws.bedrock.model.claude.postPrompt=Assistant: aws.bedrock.model.claude.maxTokens=100
```

Comment:
Line break. Each key-value pair should be on a separate line for readability purposes.